ui-kit.code-workspace

```diff
@@ -0,0 +1,123 @@
+{
+  "folders": [
+    {
+      "name": "✨ ui-kit",
```

Comment:
Is this to access files at the root level? Would it include all packages etc as well? Or could packages be excluded here for simplicity?

Reply:
Yes, it's to access files at the root level. It excludes the `packages` directory.